themes/digital.gov/src/js/common/subscribe.js

```diff
@@ -0,0 +1,9 @@
+// scroll form into view when on /subscribe page
+const subscribeButton = document.querySelector(".subscribe-button");
+const signupForm = document.querySelector("#newsletter-signup ");
```

Comment:
Let's remove the extra space here.

```suggestion
const signupForm = document.querySelector("#newsletter-signup");
```